### PR TITLE
🌐(backend) update translations

### PR DESCRIPTION
**Pull request by madmatah**
## Purpose

A message has been replaced in the widget UI to avoid repetition.

In addition to that, translations have been updated from crowdin.

### CODE REVIEW

**Comment by madmatah** [comment]
Sorry I put this PR in WIP because i'm translating a new batch of messages on crowdin.

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -120,40 +120,40 @@ msgid "Permissions successfully applied"
...
-msgstr ""
+msgstr "Authentifié"
 
 msgid "Please select this option if you want to edit the permissions of every (non-specific) logged in user"
-msgstr ""
+msgstr "Veuillez sélectionner cette option si vous souhaitez modifier les permissions de tous des utilisateurs authentifiés"
 
 msgid "Choose either a user ID or check either the anonymous or authenticated user checkbox"
-msgstr ""
+msgstr "Choisissez un identifiant utilisateur ou bien cochez la case utilisateurs authentifiés ou utilisateurs anonymes"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:138]: Should it be in the plural form?

```suggestion
msgstr "Choisissez un identifiant utilisateur ou bien cochez la case utilisateur authentifié ou utilisateur anonyme"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -204,52 +204,52 @@ msgid "A topic can not be associated with a category or a link forum"
...
 msgid "Posts"
 msgstr "Messages"
 
 msgid "A user id or an anonymous key must be associated with a post."
-msgstr ""
+msgstr "Un identifiant d'utilisateur ou une clé anonyme doit être associé à un message."
 
 msgid "A user id or an anonymous key must be associated with a post, but not both."
-msgstr ""
+msgstr "Un message doit être associé à identifiant utilisateur ou à une clé anonyme, mais pas aux deux."
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:237]:
```suggestion
msgstr "Un message doit être associé à un identifiant d'utilisateur ou à une clé anonyme, mais pas aux deux."
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -258,52 +258,52 @@ msgid "Post topic as"
...
-msgstr ""
+msgstr "Commentaire"
 
 msgid "Attachment"
-msgstr ""
+msgstr "Pièce jointe"
 
 msgid "Attachments"
-msgstr ""
+msgstr "Fichiers joints"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:297]: Or:
```suggestion
msgstr "Piéces jointes"
```

To be consistent with the previous translation.

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -258,52 +258,52 @@ msgid "Post topic as"
...
-msgstr ""
+msgstr "Indiquez la durée du sondage (nombre de jours)"
 
 msgid "Enter 0 or leave blank for a never ending poll."
-msgstr ""
+msgstr "Entrez 0 ou laisser le champ vide si vous souhaitez que le sondage soit permanent."
 
 msgid "Allow re-voting?"
-msgstr ""
+msgstr "Autoriser le changement de vote ?"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:279]: Changement de vote ou double vote ?

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -312,73 +312,73 @@ msgid "Topic polls"
 msgstr "Sondages"
 
 msgid "Poll"
-msgstr ""
+msgstr "Sondage"
 
 msgid "Poll option text"
-msgstr ""
+msgstr "Text du choix proposé"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:318]:
```suggestion
msgstr "Texte du choix proposé"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -312,73 +312,73 @@ msgid "Topic polls"
...
-msgstr ""
+msgstr "Vote du sondage"
 
 msgid "Topic poll votes"
-msgstr ""
+msgstr "Votes du sondage"
 
 msgid "A user id or an anonymous key must be used."
-msgstr ""
+msgstr "Un identifiant utilisateur ou une clé anonyme doit être utilisée."
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:342]:
```suggestion
msgstr "Un identifiant utilisateur ou une clé anonyme doit être utilisé."
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -312,73 +312,73 @@ msgid "Topic polls"
...
 
 msgid "A user id or an anonymous key must be used, but not both."
-msgstr ""
+msgstr "Un identifiant utilisateur ou une clé anonyme doit être utilisée, mais pas les deux."
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:345]:
```suggestion
msgstr "Un identifiant utilisateur ou une clé anonyme doit être utilisé, mais pas les deux."
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -399,16 +399,16 @@ msgid "Total posts"
...
-msgstr ""
+msgstr "Profil"
 
 msgid "Forum profiles"
-msgstr ""
+msgstr "Profils"
 
 msgid "Machina: Forum members"
-msgstr ""
+msgstr "Machina: utilisateurs"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:408]:
```suggestion
msgstr "Machina : Utilisateurs"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -447,52 +447,52 @@ msgid "Would you want to change this topic to an announce?"
 msgstr "Voulez-vous transformer cette discussion en annonce ?"
 
 msgid "This post has been approved successfully."
-msgstr ""
+msgstr "Cette publication a été approuvé avec succès."
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:450]:
```suggestion
msgstr "Cette publication a été approuvée avec succès."
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -447,52 +447,52 @@ msgid "Would you want to change this topic to an announce?"
...
-msgstr ""
+msgstr "Permission du forum"
 
 msgid "Has perm"
-msgstr ""
+msgstr "A la permissions"
 
 msgid "Target anonymous user"
-msgstr ""
+msgstr "Cible tout utilisateur anonymes"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:465]:
```suggestion
msgstr "Cible tout utilisateur anonyme"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -447,52 +447,52 @@ msgid "Would you want to change this topic to an announce?"
...
-msgstr ""
+msgstr "Permission de groupe"
 
 msgid "Group forum permissions"
-msgstr ""
+msgstr "Permissions de groupe"
 
 msgid "Machina: Forum permissions"
-msgstr ""
+msgstr "Machina: Permissions"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:489]:
```suggestion
msgstr "Machina : Permissions"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 12]
```diff
@@ -546,79 +546,81 @@ msgid "Can add posts in locked topics"
 msgstr "Peut ajouter des messages dans des discussions verrouillées"
 
 msgid "Machina: Forum searches"
-msgstr ""
+msgstr "Machina: Recherche"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:549]:
```suggestion
msgstr "Machina : Recherche"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 13]
```diff
@@ -546,79 +546,81 @@ msgid "Can add posts in locked topics"
...
-msgstr ""
+msgstr "Suivi d'une discussion"
 
 msgid "Topic tracks"
-msgstr ""
+msgstr "Suivis des discussions"
 
 msgid "Machina: Forum tracking"
-msgstr ""
+msgstr "Machina: Suivi des discussions"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:588]:
```suggestion
msgstr "Machina : Suivi des discussions"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 14]
```diff
@@ -546,79 +546,81 @@ msgid "Can add posts in locked topics"
...
-msgstr ""
+msgstr "Le paramètre MACHINA_MARKUP_LANGUAGE est obligatoire"
 
 msgid "Files of size greater than {} are not allowed. Your file is {}"
-msgstr ""
+msgstr "Les fichiers d'une taille supérieure à {} ne sont pas autorisés. La taille de votre fichier est de {}"
 
 msgid "Images of width lesser than {}px or greater than {}px or are not allowed. The width of your image is {}px"
-msgstr ""
+msgstr "Les images de largeur inférieur à {} px ou supérieur à {} px ne sont pas autorisées.\n"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:618]:
```suggestion
msgstr "Les images de largeur inférieure à {} px ou supérieure à {} px ne sont pas autorisées.\n"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 15]
```diff
@@ -546,79 +546,81 @@ msgid "Can add posts in locked topics"
...
+"La largeur de votre image est de {} px"
 
 msgid "Images of height lesser than {}px or greater than {}px or are not allowed. The height of your image is {}px"
-msgstr ""
+msgstr "Les images de hauteur inférieur à {} px ou supérieur à {} px ne sont pas autorisées.\n"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:622]:
```suggestion
msgstr "Les images de hauteur inférieure à {} px ou supérieure à {} px ne sont pas autorisées.\n"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 16]
```diff
@@ -654,61 +656,61 @@ msgid "Link:"
 msgstr "Lien :"
 
 msgid "Edit user permissions"
-msgstr ""
+msgstr "Modifier les permissions de l'utilisateur"
 
 msgid "Edit authenticated user permissions"
-msgstr ""
+msgstr "Modifier les permissions des utilisateur authentifiées"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:662]:
```suggestion
msgstr "Modifier les permissions des utilisateurs authentifiées"
```

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po [hunk 17]
```diff
@@ -654,61 +656,61 @@ msgid "Link:"
...
-msgstr ""
+msgstr "Accueil"
 
 msgid "Permission"
-msgstr ""
+msgstr "Permission"
 
 msgid "Save"
-msgstr ""
+msgstr "Sauvergarder"
```
Comment (jmaupetit) [inline src/ashley/locale/fr_FR/LC_MESSAGES/django.po:671]:
```suggestion
msgstr "Sauvegarder"
```

**Review by jmaupetit — COMMENTED** [review]
Thank you for taking time to translate this app :pray:

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po (hunk 4 above)
Comment (madmatah) [reply src/ashley/locale/fr_FR/LC_MESSAGES/django.po:279]: C'est pour changer de vote.
Le texte d'aide associé à ce champ de formulaire est `If enabled users are able to change their vote.`

**Comment by madmatah** [comment]
@jmaupetit Thank you very much for your proofreading :pray: 
I've corrected the errors you found.

File: src/ashley/locale/fr_FR/LC_MESSAGES/django.po (hunk 4 above)
Comment (jmaupetit) [reply src/ashley/locale/fr_FR/LC_MESSAGES/django.po:279]: Ok then.

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**